.github/workflows/docker-push.yml

```diff
@@ -1,6 +1,12 @@
 ---
 name: "Dockerfile Push"
 on:  # yamllint disable-line rule:truthy
+  pull_request:
+    paths:
+      - .devcontainer/Dockerfile
+      - .github/workflows/docker-push.yml
+      - lte/gateway/docker/mme/Dockerfile.ubuntu20.04
+      - lte/gateway/docker/python-precommit/Dockerfile
```

Comment:
This part was only inserted to trigger a new devcontainer image build, however, [it seems that only master branch builds are allowed to push new images to GHCR](https://github.com/magma/magma/runs/4153376728?check_suite_focus=true#step:6:23662).

Reply:
Yes you'll have to push the devcontainer changes and then update the hash.